templates/privacy-policy.html

```diff
@@ -0,0 +1,103 @@
+{% endblock %}
+
+{% block og_params %}
+    <!-- OG params for sharable content -->
+    <meta property="og:type" content="website">
+    <meta property="og:url" content="https://whotracks.me">
+    <meta property="og:title" content="WhoTracks.me - Privacy Policy">
+    <meta property="og:description" content="">
+    <meta property="og:image" content="https://whotracks.me/{{PATHS.static|absolute_og_urls}}/img/tracking-matrix-bluebg.png">
+{% endblock %}
+
+{% set active_page='home' %}
+
+
+{% block content %}
+<div style="height: 100vh;">
+    <div class="container">
+
+    <h1>Privacy Policy</h1>
+    <br />
+    <br />
+    WhoTracks.Me is a Cliqz GmbH project.
+    <br />
+    <br />
+
+    <div class="row">
+        <div class="col-md-6 col-sm-6 col-xs-12 menu">
+            <p>Cliqz GmbH</p>
+
+            <p>Arabellastrasse 23</p>
+            <p>81925 Munich, Germany</p>
+            <p>phone: +49 89 9250 1055</p>
+            <p>email: info[at]cliqz[dot]com</p>
+            <p>internet: www.cliqz.com</p>
+        </div>
+        <div class="col-md-6 col-sm-6 col-xs-12 menu">
+            Register court: Amtsgericht Munich <br />
+            Trade register number: HRB 173779 <br />
+            Managing directors: Dr. Marc Al-Hames, Jean-Paul Schmetz <br />
+            Value-added tax identification number, in accordance with § 27 of the Umsatzsteuergesetz: DE 260117842<br />
+            Responsible in terms of content, in accordance with § 55 Abs. 2 RStV: Jean-Paul Schmetz (address as above)<br />
+            Liability note: Despite careful content control we assume no liability for the content of external links. <br />
+            For the corresponding content of linked pages, their administrators are responsible.<br />
+        </div>
+    </div>
+
+
+    <h2>Scope of validity</h2>
+
+    This data privacy statement informs users of the nature, scope and purpose of the
+    collection and use of personal data on this website by Cliqz GmbH
+    (hereinafter: the “Provider”), Arabellastr. 23, 81925 Munich, Germany;
+    email: info[at]cliqz[dot]com, phone +49 89 9250 1055.
+
+    The provisions of the German Federal Data Protection Act (BDSG) and the German
+    Telemedia Act (TMG) form the legal basis for data privacy. Our Data protection Officer
+    is:
+    <br />
+    <br />
+
+    Mr Andreas Schmidt, LL.M. <br />
+    tekit Consult Bonn GmbH <br />
+    TÜV Saarland Gruppe<br />
+    Alexanderstraße 10, 53111 Bonn<br />
+    Geschäftsführung T. Greiner<br />
+    USt-IdNr.: DE 210856888<br />
+    Amtsgericht Bonn HRB 9029<br />
+    email: Andreas.Schmidt@tekit.tuev-saar.de<br />
+
+
+    <h2>Access data / server log files</h2>
+
+    We log requests to our website via
+    <a href="https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/AccessLogs.html">
+    CloudFront's logging mechanism.</a> Each day, a script processes these logs, to derive
```

Comment:
"We log requests to this websites via CloudFront's logging mechanism. These logs, which contain visitors' IP addresses, are anonymised daily with a process which encrypts this IP with a key which is subsequently destroyed. After this step, the original IP addresses can not longer be recovered. The original CloudFront logs are deleted after 24 hours.

In summary, access logs containing IP addresses are kept for 24 hours. Anonymised logs which are held longer have no identifiers, and cannot be linked to the requestor IP address."